docs/analysis-workflows/data-types.md

```diff
@@ -0,0 +1,176 @@
+| \*.duplicates_metrics.tgz | Generated by biobambam2 bammarkduplicates2 | qc_metrics    | Quality Control Metrics | DNA Seq Alignment      |
+
+### OxoG Metrics
+
+Data file containing OxoG metrics, metrics quantifying the error rate resulting from oxidative artifacts, for aligned CRAM files.
+
+#### File Types
+
+| Filename Pattern    | Description                          | Analysis Type | Data Category           | Generating Workflow(s) |
+| ------------------- | ------------------------------------ | ------------- | ----------------------- | ---------------------- |
+| \*.oxog_metrics.tgz | Generated by GATK CollectOxoGMetrics | qc_metrics    | Quality Control Metrics | DNA Seq Alignment      |
+
+### Ploidy and Purity Estimation
+
+Data file containing purity and ploidy estimate.
+
+#### File Types
+
+| Filename Pattern     | Description                          | Analysis Type | Data Category           | Generating Workflow(s)     |
+| -------------------- | ------------------------------------ | ------------- | ----------------------- | -------------------------- |
+| \*.ascat_metrics.tgz | Estimated by Sanger ASCAT CNV caller | qc_metrics    | Quality Control Metrics | Sanger WGS Variant Calling |
+
+### Cross Sample Contamination
+
+Data file containing cross sample contamination estimate.
+
+#### File Types
+
+| Filename Pattern             | Description                                                                                                                                     | Analysis Type | Data Category           | Generating Workflow(s)     |
+| ---------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------- | ------------- | ----------------------- | -------------------------- |
+| \*.contamination_metrics.tgz | Estimated by Sanger `verifyBamHomChk.pl` script, which provides information to determine whether the sample is possibly contaminated or swapped | qc_metrics    | Quality Control Metrics | Sanger WGS Variant Calling |
+
+### Genotyped Gender Metrics
+
+Data file containing genotypes comparison results of CRAM files from the same donor, including the fraction of matched genotypes and inferred the donor gender.
+
+#### File Types
+
+| Filename Pattern                | Description                                          | Analysis Type | Data Category           | Generating Workflow(s)     |
+| ------------------------------- | ---------------------------------------------------- | ------------- | ----------------------- | -------------------------- |
+| \*.genotyped_gender_metrics.tgz | Generated by Sanger `compareBamGenotypes.pl` script. | qc_metrics    | Quality Control Metrics | Sanger WGS Variant Calling |
+
+### Variant Calling Supplements
+
+Data file containing variation results, temporary files or runtime metrics information generated by variant callers used in the ARGO analytic pipeline.
```

Comment:
- [ ] Data file containing variation results. These are temporary files or runtime metrics information generated by variant callers used in the ARGO Analysis pipeline.

Reply:
i fixed the commas rather than a (.)

Reply:
also changing ARGO analytic pipeline to ARGO Analysis pipeline like in the menu item label